content/en/docs/ibm/deploy/install-kubeflow.md

```diff
@@ -48,18 +50,18 @@ get the best experience from Kubeflow.
 
 3. Install the IBM Cloud Block Storage plug-in. When you install the plug-in, pre-defined block storage classes are added to your cluster.
```

Comment:
Suggestion:

- A blank line separator before a code block here and everywhere else.